e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -777,3 +764,67 @@ async function assertBackgroundImageContrast(page, expected) {
     expect(actual).toBe(expected);
 }
 
+/**
+ * Use the '+' button to zoom in. Hovers first if the toolbar is not visible
+ * and waits for the zoom animation to finish afterwards.
+ * @param {import('@playwright/test').Page} page
+ */
+async function buttonZoomIn(page) {
```

Comment:
😎
This is awesome. We should consider renaming this function to be more descriptive